src/main/java/com/hubspot/jinjava/lib/fn/Functions.java

```diff
@@ -326,11 +326,20 @@ public static PyishDate stringToDate(String dateString, String dateFormat) {
 
     try {
       String convertedFormat = StrftimeFormatter.toJavaDateTimeFormat(dateFormat);
-      return new PyishDate(
+      ZonedDateTime.of(
         LocalDate
           .parse(dateString, DateTimeFormatter.ofPattern(convertedFormat))
-          .atTime(0, 0)
-          .toInstant(ZoneOffset.UTC)
+          .atTime(0, 0),
+        getDefaultZoneId()
+      );
```

Comment:
I think you didn't mean to include this block.

Reply:
whoops 😄